packages/admin/cms-admin/src/dam/FileForm/FileSettingsFields.tsx

```diff
@@ -191,8 +208,7 @@ export const FileSettingsFields = ({ isImage, folderId }: SettingsFormProps): Re
                                                         );
                                                     }
                                                 }}
-                                                disabled={value === "NO_LICENSE"}
-                                                required={value === "ROYALTY_FREE" ? false : damConfig.requireLicense}
```

Comment:
This approach to set `required` was already buggy before. The value of `required` always lagged behind by one type. 

So now I implemented a custom `requiredValidator` instead of using this prop